docker-compose-eth-besu.yaml

```diff
@@ -15,12 +15,12 @@
 version: '3.5'
 
 services:
-  avalon-blockchain-connector:
-    container_name: avalon-blockchain-connector-eth
-    image: avalon-blockchain-connector-eth-dev
+  avalon-ethereum-connector:
+    container_name: avalon-ethereum-connector
+    image: avalon-etheereum-connector-dev
```

Comment:
Change avalon-etheereum-connector-dev to avalon-etheereum-connector-dev

Reply:
Done